dungeon/assetlist.json

```diff
@@ -0,0 +1,32 @@
+{
+  "chain_name": "dungeon",
+  "assets": [
+    {
+      "description": "Dungeon native token",
+      "denom_units": [
+        {
+          "denom": "udgn",
+          "exponent": 0
+        },
+        {
+          "denom": "DGN",
+          "exponent": 6
+        }
+      ],
+      "base": "udgn",
+      "name": "Dungeon Coin",
+      "display": "DGN",
+      "symbol": "DGN",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/Quasar-hub/Dungeon/main/images/DGN.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/Quasar-hub/Dungeon/main/images/DGN.png",
+          "theme": { "circle": true }
+        }
+      ],
+      "type_asset": "native"
```

Comment:
"native" is not an accepted value. You're probably looking for "sdk.coin", here.
https://github.com/cosmos/chain-registry/blob/master/assetlist.schema.json#L67